cli/conf.py

```diff
@@ -7,12 +7,25 @@
 from cli import logger
 
 LOG = logger.LOG
-DEFAULT_CONF_DIRS = dict(
-    settings='settings',
-    modules='library',
-    roles='roles',
-    playbooks='playbooks'
-)
+
+DEFAULT_SECTIONS = {
+    'defaults': dict(
+        settings='settings',
+        modules='library',
+        roles='roles',
+        playbooks='playbooks'
+    ),
+
+    'provisioner':  dict(
+        main_playbook='provision.yml',
```

Comment:
Nope. IR will look automatically into the playbook folder (configured in defaults section) during execution phase.